projects/demo/src/modules/components/combo-box/examples/5/index.ts

```diff
@@ -0,0 +1,263 @@
+import {Component} from '@angular/core';
+import {changeDetection} from '../../../../../change-detection-strategy';
+import {encapsulation} from '../../../../../view-encapsulation';
+
+@Component({
+    selector: 'tui-combo-box-example-5',
+    templateUrl: './index.html',
+    styleUrls: ['./index.less'],
+    changeDetection,
+    encapsulation,
+})
+export class TuiComboBoxExample5 {
+    value = null;
+
+    readonly countries = [
```

Comment:
can we reuse from shared files?

Reply:
It's just a demo example, I think it would just make it less obvious and harder to maintain in terms of Opening on StackBlitz.